api/core/model_runtime/model_providers/openrouter/llm/claude-3-5-haiku.yaml

```diff
@@ -0,0 +1,39 @@
+model: anthropic/claude-3-5-haiku
+label:
+  en_US: claude-3-5-haiku
+model_type: llm
+features:
+  - agent-thought
+  - vision
```

Comment:
Vision is not currently supported.

Reply:
@Loongphy Thanks, fixed in #10364 .. Didn't expect Haiku 3.5 to have less features than Haiku 3 :))